simulation_methods.hpp

```diff
@@ -6,6 +6,7 @@
 #include "json.hpp"
 #include <string>
 #include "vector.hpp"
+#include <chrono>
 
 using json = nlohmann::json;
 
```

Comment:
please check values inside the a matrix and explain why are they given in that way.

Reply:
well I obviously intended to store -1/2 as whole fraction like -0.5. I think "(double)-1/2" returns exactly the same as "(double)(-1/2)". And judging by the method performance the values are indeed the ones I intended to store.